public/video-ui/src/components/Video/Display.js

```diff
@@ -60,27 +87,50 @@ class VideoDisplay extends React.Component {
         <VideoSelectBar video={video} onSelectVideo={this.selectVideo} embeddedMode={this.props.config.embeddedMode} />
 
         <div className="video">
-          <div className="video__sidebar video-details">
-            <form className="form video__sidebar__group">
-              <VideoEdit
-                video={this.props.video || {}}
-                updateVideo={this.updateVideo}
-                saveVideo={this.saveVideo}
-                saveAndUpdateVideo={this.saveAndUpdateVideo}
-                resetVideo={this.resetVideo}
-                saveState={this.props.saveState}
-                disableStatusEditing={this.cannotEditStatus()}
-               />
-            </form>
-          </div>
-
           <div className="video__main">
             <div className="video__main__header">
               <div className="video__detailbox">
                 <span className="video__detailbox__header">Preview</span>
                 <VideoPreview video={this.props.video || {}} />
               </div>
-              <div className="video__detailbox usages">
+              <div className="video__detailbox">
+                <div className="video__detailbox__header__container">
+                  <span className="video__detailbox__header">Video Meta Data</span>
```

Comment:
Is a `<span>` appropriate? Marking it up as a header would make more semantic sense.